projects/demo-playwright/tests/kit/carousel/carousel.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Carousel`, () => {
+    test(`default padding`, async ({page}) => {
+        await tuiGoto(page, `components/carousel/API`);
+        const documentationPage = new TuiDocumentationPagePO(page);
+        const example = documentationPage.apiPageExample;
+
+        await expect(example).toBeVisible();
+        await expect(example).toHaveScreenshot(`carousel-padding-default.png`);
+    });
+
+    test(`padding has zero value`, async ({page}) => {
+        await tuiGoto(page, `components/carousel/API?style.--tui-carousel-padding=0`);
+        const documentationPage = new TuiDocumentationPagePO(page);
+        const example = documentationPage.apiPageExample;
+
+        await expect(example).toBeVisible();
+        await expect(example).toHaveScreenshot(`carousel-padding-0.png`);
+    });
+
+    test(`should show next item after drag`, async ({page}) => {
+        await tuiGoto(page, `components/carousel/API?draggable=true`);
+        const documentationPage = new TuiDocumentationPagePO(page);
+        const example = documentationPage.apiPageExample;
+        const itemExample = example.locator(`.t-item`).first();
```

Comment:
Just `item` / `slide` ?

```suggestion
        const item = example.locator(`.t-item`).first();
```

Reply:
fixed